src/inputs/select.scss

```diff
@@ -28,7 +28,7 @@ $iui-after-border: 5px solid;
       @include iui-menu-icon;
     }
 
-    > .iui-content {
+    > .iui-label {
```

Comment:
Why was this renamed?

Reply:
Button, select, menu, header button are now all aligned to use `.iui-icon + .iui-label`.  We could re-use the `@include iui-menu-icon` across more components now.

Reply:
I also don't like this rename. Because now we have `label` inside `label`, `content` makes more sense here.
![image](https://user-images.githubusercontent.com/36186912/124432957-dd17b880-dd7a-11eb-88a6-ddd7589aff93.png)

Reply:
Reverted back to `.iui-content` for the select component.  It should probably be renamed in another PR to more closely align with `.iui-menu-item-content`.

The menu icon mixin is now targeting `[class^='iui-'][class$='-content']` which should effect `.iui-menu-item-content`, `.iui-content` (as it is now), and `.iui-select-content` if / when we decide to rename that class.

Reply:
I reverted back to `iui-content`. I hope Jon won't be mad about this 🤞

Reply:
I'm not mad, I'm just happy this PR is finally about to pass! ☺️